### PR TITLE
Feature/roecct 198 oe name bugfix

### PR DESCRIPTION
**Pull request by cmowatt-ch**
### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by ConnorOS-ch — COMMENTED** [review]
good job these are major improvements 👍 left a few questions about certain points

File: src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts [hunk 1]
```diff
@@ -126,3 +147,11 @@ const getPagePropertiesRelevantPeriod = (relevant_period, trust, formData, trust
...
   return pageProps;
 };
+
+export const setEntityNameInRelevantPeriodPageBanner = (pageProps, entityName: string | undefined) => {
+  // name the entity for the page template
+  if (pageProps && pageProps.pageData && entityName !== undefined) {
+    pageProps.pageData.entity_name = entityName;
+  }
+  return pageProps;
+};
```
[the remark this answers is not in the document]
Comment (cmowatt-ch) [reply src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts]: As discussed, the unit/integration test execution makes defending against undefined necessary. Refactoring to use setEntityNameInRelevantPeriodPageBanner in the other places for consistency.

**Review by ConnorOS-ch — APPROVED** [review]
Looks good, thanks for addressing my comments

**Comment by amalviyach** [comment]
Ticket has been tested on ci-dev closing this one to reduce confusion.

**Pull request closed without merging**